ApplyToBecomeInternal/ApplyToBecomeInternal.Tests/Models/ApplicationForm/Sections/FinanceSectionTests.cs

```diff
@@ -171,6 +171,84 @@ public void Constructor_Doesnt_Set_Conditional_Rows_Following_No_Answers_And_Sur
 				new FormField("Are there any financial investigations ongoing at the school?", "No")
 			};
 
+			formSection.Heading.Should().Be("Finances");
+			formSection.SubSections.Should().HaveCount(6);
+			var subSections = formSection.SubSections.ToArray();
+			subSections[0].Heading.Should().Be("Previous financial year");
+			subSections[0].Fields.Should().BeEquivalentTo(expectedPreviousFinancialYearFields);
+			subSections[1].Heading.Should().Be("Current financial year");
+			subSections[1].Fields.Should().BeEquivalentTo(expectedCurrentFinancialYearFields);
+			subSections[2].Heading.Should().Be("Next financial year");
+			subSections[2].Fields.Should().BeEquivalentTo(expectedNextFinancialYearFields);
+			subSections[3].Heading.Should().Be("Loans");
+			subSections[3].Fields.Should().BeEquivalentTo(expectedLoansFields);
+			subSections[4].Heading.Should().Be("Financial leases");
+			subSections[4].Fields.Should().BeEquivalentTo(expectedFinancialLeasesFields);
+			subSections[5].Heading.Should().Be("Financial investigations");
+			subSections[5].Fields.Should().BeEquivalentTo(expectedFinancialInvestigationsFields);
+		}
+		[Fact]
+		public void Constructor_Doesnt_Set_Conditional_Rows_Following_Yes_Answers_And_Surplus_Answers()
```

Comment:
I don't think the title of this test method is correct. It seems to be testing that the conditional rows ARE set following DEFICIT answers